docs/api/web-frame.md

```diff
@@ -314,3 +314,8 @@ current renderer process.
 An `Integer` representing the unique frame id in the current renderer process.
 Distinct WebFrame instances that refer to the same underlying frame will have
 the same `routingId`.
+
+### `webFrame.windowFeatures` _Readonly_
+
+A `String` representing the encoded list of window features passed into
+`window.open(url, target, windowFeatures)`.
```

Comment:
This is much too concise! Please expand this to make it clearer when this is present, what happens upon navigation, etc.